Cargo.toml

```diff
@@ -25,9 +29,12 @@ path = "./benches/search.rs"
 test = false
 bench = true
 
+[dependencies.memmap]
+version = "0.4.0"
+optional = true
```

Comment:
Can you write this as `memmap = { version = "0.4.0", optional = true }`? I prefer to keep all deps in one place for easy reading if possible.